bop_toolkit_lib/tests/test_misc.py

```diff
@@ -41,10 +43,14 @@ def test_transform_pts_Rt(self):
     def test_project_pts(self):
 
         """
-        For this test, we cannot sample poses completely at random since
-        Points close to the camera plane Zc~0 will create numerical issues.
-        Instead, sample camera rotation and translation around
-        a reference pose that looks toward the point cloud.
+        Compare 3 pinhole projection implementations:
```

Comment:
Did you verify the `CameraModelType.FISHEYE624` as well? I know it is harder to test, but maybe you have one example where you know the image-3D correspondences.

Reply:
Not as a unittest indeed, I'm not sure how we could produce such an example

Reply:
But you have visualized the results right?